test/ex_machina/ecto_test.exs

```diff
@@ -42,4 +42,18 @@ defmodule ExMachina.EctoTest do
       TestFactory.params_for(:user_map)
     end
   end
+
+  test "params_with_assocs/2 inserts belongs_to associations" do
+    assert TestFactory.params_with_assocs(:article) == %{
+      title: "My Awesome Article",
+      author_id: ExMachina.TestRepo.first!(User).id,
+    }
+  end
+
+  test "params_with_assocs/2 inserts associated fields" do
```

Comment:
I'm not sure I understand this test. What is this covering that the previous one isn't?